bug(linux): deploy to llso failing

https://github.com/keymanapp/keyman/actions/runs/7759768180/job/21164843272

```
/runner/_work/_temp/a752876d-acdf-444f-8f24-58460[62](https://github.com/keymanapp/keyman/actions/runs/7759768180/job/21164843272#step:7:63)1e902.sh: line 37: cd: keyman-srcpkg: No such file or directory
  Error: Process completed with exit code 1.
```

--- a/.github/workflows/deb-packaging.yml
+++ b/.github/workflows/deb-packaging.yml
@@ -241,15 +241,12 @@ jobs:
         esac
 
         echo -e "::group::${COLOR_GREEN}Upload source package"
-        cd keyman-srcpkg
-
         dput_retry -U llso:ubuntu/jammy${destination} *_source.changes
         echo "::endgroup::"
 
         echo -e "::group::${COLOR_GREEN}Uploading binary packages"
-        cd ../keyman-binarypkgs
         pattern="keyman_${{ needs.sourcepackage.outputs.VERSION }}.+\+(.*)[0-9]_[^.]+.changes"
-        for f in *.changes; do
+        for f in *_amd64.changes; do
           if [[ $f =~ $pattern ]]; then
             dist=${BASH_REMATCH[1]}
             dput_retry -U llso:ubuntu/${dist}${destination} $f

